### PR TITLE
chore: bump start-stack to 2023.09.01 (DEV-2630)

### PR DESCRIPTION
**Pull request by Nora-Olivia-Ammann**

### CODE REVIEW

File: src/dsp_tools/resources/start-stack/docker-compose.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ services:
   api:
     # on the verge of every deployment (fortnightly), update the "image" value from the "api" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/roles/dsp-deploy/files/RELEASE.json
-    image: daschswiss/knora-api:29.1.3
+    image: daschswiss/knora-api:v30.0.0
```
Comment (jnussbaum) [inline src/dsp_tools/resources/start-stack/docker-compose.yml:52]: `knora-sipi` (line 31) must always have the same value than `knora-api`. We need to bump them both to the same version number

File: src/dsp_tools/resources/start-stack/start-stack-config.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 ---
 
 # take commit hash of latest DSP-API release from https://github.com/dasch-swiss/dsp-api/commits/main
-DSP-API commit: c95b146bcc75828b84619b9e158d10353934fcab
+DSP-API commit: f124f855043443d355a6d07b6fb384361c7b4e24
```
Comment (jnussbaum) [inline src/dsp_tools/resources/start-stack/start-stack-config.yml:4]: This is the latest commit of DSP-API, but not the latest release. What we need is the commit hash of the commit "chore(main): release 30.0.0"

Comment (Nora-Olivia-Ammann) [reply src/dsp_tools/resources/start-stack/start-stack-config.yml:4]: alles klar danke

File: src/dsp_tools/resources/start-stack/docker-compose.yml (hunk 1 above)
Comment (Nora-Olivia-Ammann) [reply src/dsp_tools/resources/start-stack/docker-compose.yml:52]: ok then i misunderstood that

**Review by jnussbaum — APPROVED** [review]

**Pull request merged**